setup.py

```diff
@@ -413,7 +413,7 @@ def run(self):
               "pip==22.0.3",
               "wheel==0.37.1",
               "setuptools==60.9.1",
-              "subunitreporter==22.2.0",
+              "subunitreporter==23.8.0",
```

Comment:
subunitreporter 22.2.0 has unicode handling problems on Windows that were revealed here.  23.8 fixes them.

Reply:
I also pulled this upgrade out into its own PR - #1329 - since it has consequences for the Docker images :/